DBlock Game/DBlock Game/Program.cs

```diff
@@ -1275,6 +1275,272 @@ public static void BreakRoom()
                 }
             }
         }
+        public static void KylesRoom()
+        {
+            //This section is the text a player gets from entering the room and sets up the choices 
+            string temp;
+            char userinput;
+            loading();
+            TextBox("You push open the door to the room and are met with a blinding light", Player, false);
+            TextBox("A voice bellows at you with tremendous force...", Player, false);
+            TextBox("YOU HAVE ENTERED MY ROOM MORTAL, NOW ANSWER MY QUESTIONS THREE OR FACE YOUR DOOM!!", Player, false);
+            TextBox("You step forward and the light dims down, revealing a giant Sphinx with Donnas face", Player, false);
+            TextBox("The Sphinx points towards a table with three pieces of parchment ", Player, false);
+            TextBox("There is a shelf with random objects on one side of the room and a small dressing table with a figurine on it to the right", Player, false);
+            TextBox("What will you do? Press S to explore the shelf, T to explore the small tabel and P to look at the riddles", Player, false);
+
+
+            temp = Console.ReadLine();
+            userinput = Convert.ToChar(temp);
+            //This is a case/switch statement for the player to choose an option to explore the room or just do the riddles 
+            switch (userinput)
```

Comment:
do userinput.ToUpper()

and then make every case statement a capital letter

Reply:
Not sure how to code that bit in, could you help explain where the userinput.ToUpper() bit goes? Ive tried putting it in the code and cant brain the answer haha